libass/ass.c

```diff
@@ -767,18 +900,25 @@ static bool detect_legacy_conv_subs(ASS_Track *track)
 
 static int process_events_line(ASS_Track *track, char *str)
 {
+    int ret = 0;
```

Comment:
Why is that even needed? `ret` is zero everywhere except a couple of lines.

Reply:
It's a complication for no reason, direct violation of KISS principle. I've suggested to replace return value of `parse_format_line()` to `bool` exactly to keep it conceptually correct.

Reply:
`ret` removed as it's currently only set in one place which (currently) is always a fatal error.